dev-util/cproto/cproto-4.7e.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2018 Gentoo Foundation
+# Copyright 1999-2019 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=0
 
-MY_PV="4_7e"
+MY_PV="4.7e"
 MY_P=${PN}-${MY_PV}
 S=${WORKDIR}/${MY_P}
 
 DESCRIPTION="Generate C function prototypes from C source code"
-HOMEPAGE="http://invisible-island.net/cproto/"
-SRC_URI="ftp://invisible-island.net/cproto/${MY_P}.tgz"
+HOMEPAGE="https://invisible-island.net/cproto/"
+SRC_URI="ftp://ftp.invisible-island.net/cproto//${MY_P}.tgz"
```

Comment:
why the double slash?

Reply:
1. remove `|| die` on dobin in `src_install`
2. try and replace the `dodoc` by `einstalldocs`, it should catch both files

Reply:
Well the double slash is needed. Otherwise wget won't find the file. I'm not exactly sure why, all i could find about this was [1]:

I've also didn't remove ```|| die``` and use ```einstalldocs```. Please remember this ebuild is still at ```EAPI=0```. I've only fixed the SRC_URI and use https as i would usually have todo a revision bump in order to bump ```EAPI```. - Which is my i made a version bump, because I hope to stablize the newer version soon in order to get rid of the ```EAPI=0```.
However, if you want i can make a revision bump of the existing version too.

[1]https://unix.stackexchange.com/questions/474642/using-wget-on-a-directory-outside-the-users-home-directory

Reply:
Oh yes, sorry, missed that